src/components/PortfolioChart.js

```diff
@@ -22,7 +22,7 @@ export class PortfolioChart extends Component {
         this.state.endDate._d
       );
     } else {
-      const today = DateUtils.getToday();
+      const today = DateUtils.convertToPickedDate(new Date());
       let startDate = new Date();
       startDate.setHours(12, 0, 0, 0);
```

Comment:
You have a converToPickedDate (which should be renamed) -- why aren't you using it here?

Reply:
? i am using it?